sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/SelectByteBuddyHelpers.java

```diff
@@ -256,6 +260,10 @@ public StackManipulation append(StackManipulation valueToWrite) {
       return store(currentArrayField++, valueToWrite);
     }
 
+    public int reserveSlot() {
```

Comment:
nit: (slightly related to this PR) can we make `SelectByteBuddyHelpers` package private please

Reply:
done